goverge/main.py

```diff
@@ -71,13 +71,11 @@ def get_test_packages(project_root, ignore):
     ignores = ["/.", "Godeps", "vendor"]
     if ignore is not None:
         ignores.extend(ignore)
-    directories = [project_root]
-    for root, subdirs, files in os.walk(project_root):
-        for subdir in subdirs:
-            full_path = os.path.join(root, subdir)
-            if str(full_path).startswith(project_root) and full_path not in ignores and \
-                    subdir not in ignores and full_path != project_root:
-                directories.append(os.path.join(root, subdir))
+    directories = []
+    for root, subdirs, file in os.walk(project_root):
```

Comment:
You have shadowed a builtin python variable with file and I don't think it's really accurate anyways since os.walk() returns a list of files not a single file.